tests/test_models.py

```diff
@@ -351,7 +347,7 @@ def test_patching(self):
 
 
 rates_is_available = pytest.mark.skipif(
-    not AUTO_CONVERT_MONEY,
+    not RATES_INSTALLED,
```

Comment:
Yeah, absolutely right.. something seems to have been renamed... added in 9c88515